src/Mangata.ts

```diff
@@ -113,37 +123,33 @@ export class Mangata {
    */
 
   public async createPool(
-    account: KeyringPair | string,
-    firstAssetId: string,
-    firstAssetAmount: BN,
-    secondAssetId: string,
-    secondAssetAmount: BN,
+    account: MangataAccount,
```

Comment:
Here and later: what is the reason for using substrate-provided types? Shouldn't we expose the simplest possible interface to the SDK users?

Definitely we should not use Mangata Account since confusing for the users.

Substrate interaction learning curve is extremely steep, and the goal of SDK is not to support this but help faster onboarding.

Reply:
MangataAccount is **KeyringPair | string**  type.. it is out created type.. it has nothing to do with substrate... in order to not mix substrate here I will refactor this to **Account** instead of **MangataAccount**

Reply:
But types of the following arguments are substrate-provided:
```
firstTokenId: TokenId,
firstTokenAmount: Amount,
secondTokenId: TokenId,
secondTokenAmount: Amount,
```

Can we stick to `KeyringPair | string` here? It is self-explanatory and has a semantic sense since this is not a single type but two different ones by design.

Reply:
I will refactor it back